Code/Engine/Texture/TexConv/Implementation/TextureModifications.cpp

```diff
@@ -26,7 +26,7 @@ ezResult ezTexConvProcessor::ForceSRGBFormats()
   return EZ_SUCCESS;
 }
 
-ezResult ezTexConvProcessor::GenerateMipmaps(ezImage& img, ezUInt32 uiNumMips, MipmapChannelMode channelMode) const
```

Comment:
This is not correct and overwriting a change made on the ez side

Reply:
fixed